memory-db/src/lib.rs

```diff
@@ -18,6 +18,8 @@
 #![cfg_attr(not(feature = "std"), feature(alloc))]
 
 extern crate hash_db;
+extern crate parity_util_mem as malloc_size_of;
```

Comment:
Personal opinion but I think the rename just makes things confusing.

Reply:
I agree, I must have use the derive macro at some point (that is the kind of line that is required by this macro).